bin/ledger2beancount

```diff
@@ -736,7 +736,7 @@ while (@input) {
     my @stanza;
     if ($l =~ /^[!@]?include\s+(?<filename>.*)\.ledger/) {  # include
 	print_line $depth, "include \"$+{filename}.beancount\"";
-    } elsif ($l =~ /^$comment_top_level_RE/) {
+    } elsif ($l =~ /$comment_top_level_RE/) {
```

Comment:
Can we keep the heading regex `^` visible here, in the main conversion loop, rather than moving it to the regex variable? Currently all other regexs do that, and it's surprising not seeing it for a single regex, making the code less readable.